inc/datahandlers/post.php

```diff
@@ -1018,7 +1018,7 @@ function insert_post()
 			}
 		}
 
-		if($visible == 1 && $thread['visible'] == 1)
+		if($visible == 1)
```

Comment:
This appears to be incorrect. If the thread isn't visible the post count shouldn't be changed.